code/__HELPERS/game.dm

```diff
@@ -485,6 +485,59 @@
 			++i
 	return L
 
+/proc/pollCultists(var/mob/living/Nominee) // Cult Master Poll
+	var/time_passed = world.time
+	set waitfor = FALSE
+	var/list/yes_voters = new
+	var/list/cult_total = new
+	if(world.time<1800)
+		Nominee << "It would be premature to select a leader while everyone is still settling in, try again in [(1800-world.time)/10] seconds"
+		return
+	for(var/datum/mind/B in SSticker.mode.cult)
+		if(!istype(B.current, /mob/living))
+			continue
+		var/mob/living/M = B.current
+		if(isliving(M))
+			M << 'sound/misc/notice2.ogg' //Alerting them to their consideration
+			M.verbs -= /mob/living/proc/cult_master
+			M << "<span class='cultlarge'>[usr] has asserted that they are worthy of leading the cult. A vote will be called shortly.</span>"
+			spawn(200)

```

Comment:
I know, I coded this shortly after I returned and have learned much since then. But I will say that spawn does work like this. They delay is there and working as intended.